website/docs/reference/resource-configs/bigquery-configs.md

```diff
@@ -690,6 +690,10 @@ from {{ ref('events') }}
 By default, dbt-created tables never expire. You can configure certain model(s)
 to expire after a set number of hours by setting `hours_to_expiration`.
 
+:::info Note
+The `hours_to_expiration` only applies to initial creation of the underlying table. For incremental models, the `hours_to_expiration` is **not** reset on incremental runs.
```

Comment:
great addition, Doug! added a non blocking suggestion @dbeatty10 to simplify the language. 

```suggestion
The `hours_to_expiration` only applies to initial creation of the underlying table. It doesn't reset for incremental models when they do another run.
```

Reply:
Love it, thank you @mirnawong1 🤩